README.md

```diff
@@ -1,4 +1,4 @@
-# react-native-network-state
+# react-native-network-state-connect-internet
```

Comment:
@linhndbtc-studio Do you have explanation for this line?
Doesn't it completely switch to new npm package which is not relevant to this repo?